### DISCUSSION TITLE
Generate eslint flat config for new workspace

### DISCUSSION THREAD

**Discussion by robertIsaac** [nrwl/nx#26338] [Feature Requests]
Nx should generate new workspaces with the new flat configuration for eslint
And it's even better to use eslint 9 with typescript-eslint@rc-v8

**Comment by codinronan** [comment]
@robertIsaac I generated a new project over the weekend with nx 19.8 and got a flat config using eslint 9 -- should be what you were asking for?

**Comment by robertIsaac** [comment]
yes this is done now
